workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -17,18 +19,28 @@ public class GdkToolsConfiguration
         public string DevAuthTokenDir;
         public int DevAuthTokenLifetimeDays;
         public bool SaveDevAuthTokenToFile;
+        public int SimulatorNameId;
```

Comment:
should this an editor pref / player pref?

Reply:
probably. might refactor this as I'm not happy with it at all right now